torch/csrc/jit/passes/python_print.cpp

```diff
@@ -368,7 +367,7 @@ struct PythonPrintPass {
       std::unordered_set<std::string>& used) {
     std::string name = candidate;
     while (used.count(name) || reserved_names.count(name)) {
-      name = candidate + std::to_string(next_id[name]++);
+      name += std::to_string(next_id[name]++);
```

Comment:
Why was this loop changed? if the while loop executes multiple times this does something different now

Reply:
this was originally complained by clang-tidy, i did some investigation and thought they are syntax equivalent for string references, i will double check on that